lisk/build/target/etc/rebuild.json

```diff
@@ -1,10 +1,10 @@
 {
 	"httpPort": 9000,
 	"wsPort": 9001,
-	"logFileName": "logs/lisk_snapshot.log",
+	"logFileName": "logs/lisk_rebuild.log",
 	"fileLogLevel": "info",
 	"db": {
-		"database": "lisk_snapshot"
+		"database": "lisk_rebuild"
```

Comment:
@michielmulders That change required to align changes in scripts as well. Please discuss with @fchavant and create respective issue for it.

Reply:
Aligned with @fchavant , `lisk/build/target/etc/rebuild.json` and `lisk/build/target/etc/pm2-rebuild.json` can be safely removed. No further influence. Thanks!